tests/mir-opt/global_asm.rs

```diff
@@ -0,0 +1,18 @@
+// skip-filecheck
+#![feature(no_core, lang_items, rustc_attrs, decl_macro)]
```

Comment:
you don't need no-core, i'm pretty sure there's a directive for limiting this test just to platforms with asm support:

`//@ needs-asm-support`

Reply:
Yeah I guess that's fine here, because the printing logic does not change between platforms so testing only on some is probably fine. I've made the change.